File: AASC.Partner.Web/AASC.Partner.Web.Tests/Models/EmployeeViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AASC.Partner.Web.Tests.Models
{
    public class EmployeeViewModel
    {
        public Guid? Id { get; set; }

        public String JobTitle { get; set; }

        public Guid CompanyId { get; set; }

        public virtual CompanyDisplayViewModel Company { get; set; }

        public string ApplicationUserId { get; set; }

        public virtual DisplayUserBindingModel ApplicationUser { get; set; }

        public DateTime? CreatedDate { get; set; }

        public string CreatedById { get; set; }

        public virtual DisplayUserBindingModel CreatedBy { get; set; }
    }

    public class EmployeeDisplayViewModel
    {
        public Guid? Id { get; set; }

        public String JobTitle { get; set; }

        public string ApplicationUserId { get; set; }

        public string Email { get; set; }

        public string Username { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }
    }
}

File: ButtRydeClient/ButtRydeClient/app/viewModels/employeeViewModel.js
'use strict';

app.factory('employeeViewModel', function () {
    return kendo.data.Model.define({
        id: 'id',
        fields: {
            id: { type: 'string', editable: false, nullable: false },
            jobTitle: { type: 'string', editable: false, nullable: true },
            companyId: { type: 'string', editable: false, nullable: true },
            company: {
                type: 'companyViewModel',
                editable: false,
                defaultValue: {
                    id: '',
                    name: ''
                }
            },
            applicationUserId: { type: 'string', editable: false, nullable: false, defaultValue: '00000000-0000-0000-0000-000000000000' },
            applicationUser: {
                type: 'displayUserBindingModel',
                editable: false,
                defaultValue: {
                    id: '',
                    username: '',
                    firstName: '',
                    lastName: '',
                    email: ''
                }
            },
            createdDate: {
                type: 'date',
                editable: false,
                nullable: true
            },
            createdById: { type: 'string', editable: false, nullable: false, defaultValue: '00000000-0000-0000-0000-000000000000' },
            createdBy: {
                type: 'displayUserBindingModel',
                editable: false,
                nullable: true
            }
        }
    });
});